.github/workflows/debian-executable-release.yml

```diff
@@ -0,0 +1,47 @@
+name: Debian Executable Release
+on:
+  workflow_dispatch:
+    inputs:
+      formula:
+        description: 'Artifact Prefix'
+        default: twilio
+jobs:
+  debian-executable-release:
+    name: Publish for ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Getting latest tag
+        id: get-tag
+        run: |
+         git fetch --prune --unshallow
+         echo "::set-output name=TAG_NAME::$(git describe --tags $(git rev-list --tags --max-count=1))"
+      - run: |
+          make install
+          sudo $(npm bin)/oclif-dev pack:deb
```

Comment:
changed `sudo npx oclif-dev pack:deb` to `sudo $(npm bin)/oclif-dev pack:deb` (works the same when oclif-dev is installed globally). We need root access(sudo) to run this command because we are changing ownership of file in oclif/dev-cli [pack](https://github.com/oclif/dev-cli/blob/main/src/commands/pack/deb.ts#L82-L84) command. With npx sudo, we are seeing privilege issue. Not sure of the root cause.